docs/src/ReproduceWillis.jl

```diff
@@ -0,0 +1,120 @@
+import Plots
+
```

Comment:
We can use those plots for testing things, but we probably wont need them in the final documentation.